iina/MainWindowController.swift

```diff
@@ -683,7 +679,7 @@ class MainWindowController: NSWindowController, NSWindowDelegate {
       button.bezelStyle = .regularSquare
       button.isBordered = false
       button.image = buttonType.image()
-      button.action = #selector(self.toolBarButtonAction(_:))
+      button.action = #selector(self.toolBarButtonAction)
```

Comment:
To be clear: this is fine. However, the name of the selector is still `MainWindowController.toolBarButtonAction(_:)`.